typo: occurance -> occurrence

nschonni opened the pull request.

nschonni committed:
typo: occurance -> occurrence

VernonMSFT commented:
@mcleblanc The PR **passed** the acceptance checklist. Please evaluate further and approve/complete the PR if appropriate. Thanks!

mijacobs committed:
Update schema-root.md

Our convention is to use "syntax" here. If we change it to xml, we should change it everywhere. We'll investigate.

mijacobs committed:
Update schema-root.md
